.travis/test-part.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# -*- firestarter: "shfmt -i 4 -ci -w %p" -*-
+
```

Comment:
what does this comment mean?

Reply:
i use shfmt to format bash. could remove if needed